### ISSUE TITLE
remove a node in tree without deleting descendants

### ISSUE THREAD

**Issue by jvacek** [fabiocaccamo/django-treenode#40]
It would be nice to allow for behaviour where deleting a node would create separate disjoint trees. Any suggestions how this could be achieved with the current implementations? I previously had just an FK field to parent node, which had a `on_delete=SET_NULL` which worked fine for me, and I would like to replicate this behaviour here.

**Comment by fabiocaccamo** [comment]
@jvacek any update?

**Comment by jvacek** [comment]
Not yet, we pushed this one down our priority list

**Comment by fabiocaccamo** [comment]
@jvacek a possible solution could be like this:
```python
from treenode.signals import no_signals

with no_signals():
    children = node.get_children()
    for child in children:
        child.set_parent(None)
node.delete()
```
This is a generic approach, it could be done with a new model method or just adding an option like `keep_children=True` to the current `delete` method. 

What do you think about it?

**Comment by fabiocaccamo** [comment]
`node.delete(cascade=False)` available since `0.18.0` version.
